src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.tsx

```diff
@@ -43,6 +57,40 @@ function BaseSidebarScreen() {
         updateLastAccessedWorkspace(undefined);
     }, [activeWorkspace, activeWorkspaceID]);
 
+    /**
+     * Navigate to scan receipt screen after it enabling camera permission from setting
+     * This will only works for ios application because we are saving last screen only for ios
+     */
+    useEffect(() => {
```

Comment:
Why do you put this `useEffect` into this file?

Reply:
@DylanDylann other screen where I can put this useEffect is SidebarScreen.tsx. Please let me know if I am missing flow flow for this.

Reply:
Let's try in Expensify.tsx file

Reply:
@DylanDylann done